### DISCUSSION TITLE
I suggest adding chatroom function to this system, so as to make it bigger and stronger.

### DISCUSSION THREAD

**Discussion by ITITIT123** [LiveHelperChat/livehelperchat#1867] [General]
I suggest adding chatroom function to this system, so as to make it bigger and stronger.

**Comment by remdex** [comment]
Chat rooms are available for operators not visitors

**Comment by ITITIT123** [comment]
My suggestion is to provide visitors with it.

**Reply by remdex** [reply]
With your sponsorship we can do that :)
